### PR TITLE
use jupyter_sphinx v0.2.0 from conda instead of my branch

### PR DESCRIPTION
**Pull request by basnijholt**
This again needs https://github.com/python-adaptive/adaptive/pull/169. However, I have manually executed those commands from the CI.

See https://adaptive.readthedocs.io/en/test_documentation/ when the build finished.

### CODE REVIEW

**Comment by akhmerov** [comment]
Let's pin jupyter-sphinx at `<= 0.3`, and specify the thebe config like [here](https://jupyter-sphinx.readthedocs.io/en/latest/#thebelab-support). Pointing to this repo should be all it takes.

**Comment by basnijholt** [comment]
Actually, I rather pin it to a fixed version like everything else.

Is adding those lines to the config really enough to install thebelab?

Of course, it would need some buttons.

File: docs/source/conf.py [hunk 1]
```diff
@@ -83,6 +83,12 @@
 pygments_style = "sphinx"
 
 
+jupyter_sphinx_thebelab_config = {
+    "requestKernel": True,
+    "binderOptions": {"repo": "binder-examples/requirements"},
```
Comment (akhmerov) [inline docs/source/conf.py]: `  "binderOptions": {"repo": "python-adaptive/adaptive"},`

**Comment by basnijholt** [comment]
Also, how would we make sure that this gets executed https://github.com/python-adaptive/adaptive/blame/master/docs/source/tutorial/tutorial.Learner1D.rst#L13-L21

**Comment by akhmerov** [comment]
> Also, how would we make sure that this gets executed https://github.com/python-adaptive/adaptive/blame/master/docs/source/tutorial/tutorial.Learner1D.rst#L13-L21

These will become visible but the user will need to manually execute those.

**Comment by jbweston** [comment]
I updated the dependency to jupyter_sphinx 0.2.1. All the other dependencies are pinned at exact version, so I think it's fine

**Comment by jbweston** [comment]
This LGTM

**Pull request merged**